barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_circuit_builder.cpp

```diff
@@ -793,23 +806,24 @@ template <typename ExecutionTrace> void UltraCircuitBuilder_<ExecutionTrace>::pr
 
     BB_ASSERT_GT(list.variable_indices.size(), 0U);
 
-    // replace witness index in variable_indices with the real variable index i.e. if a copy constraint has been
-    // applied on a variable after it was range constrained, this makes sure the indices in list point to the updated
-    // index in the range list so the set equivalence does not fail
+    // replace witness-index in variable_indices with the corresponding real-variable-index i.e., if a copy constraint
+    // has been applied on a variable after it was range constrained, this makes sure the indices in list point to the
+    // updated index in the range list so the set equivalence does not fail
     for (uint32_t& x : list.variable_indices) {
         x = this->real_variable_index[x];
     }
-    // remove duplicate witness indices to prevent the sorted list set size being wrong!
+    // Sort `variable_indices` and remove duplicate witness indices to prevent the sorted list set size being wrong!
     std::sort(list.variable_indices.begin(), list.variable_indices.end());
     auto back_iterator = std::unique(list.variable_indices.begin(), list.variable_indices.end());
     list.variable_indices.erase(back_iterator, list.variable_indices.end());
 
     // go over variables
-    // iterate over each variable and create mirror variable with same value - with tau tag
+    // iterate over each (real) variable and create mirror variable with same value - with tau tag
```

Comment:
Nit: can you add some punctuation or capitalization to these two lines so its more clear that the second line is a new thought (I think?)